src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/sentiment/WordListSentimentDetector.scala

```diff
@@ -0,0 +1,77 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment
+
+import java.io.{File, IOError, IOException}
+import java.util.concurrent.ConcurrentHashMap
+
+import com.microsoft.partnercatalyst.fortis.spark.logging.Loggable
+import com.microsoft.partnercatalyst.fortis.spark.transforms.ZipModelsProvider
+import com.microsoft.partnercatalyst.fortis.spark.transforms.nlp.Tokenizer.tokenize
+import com.microsoft.partnercatalyst.fortis.spark.transforms.sentiment.SentimentDetector.{NEGATIVE, NEUTRAL, POSITIVE}
+
+import scala.io.Source
+
+@SerialVersionUID(100L)
+class WordListSentimentDetector(
```

Comment:
I agree that it wouldn't provide immediate value, but might be nice to think through to set us up for easier extensibility in the future.